dockstore-webservice/src/main/resources/migrations.1.15.0.xml

```diff
@@ -29,4 +29,31 @@
         <dropPrimaryKey tableName="user_entry"/>
         <addPrimaryKey columnNames="userid, entryid" constraintName="user_entry_pkey" tableName="user_entry"/>
     </changeSet>
+    <changeSet author="svonworl" id="add_everpublic">
+        <addColumn tableName="workflow">
+            <column name="everpublic" type="boolean" value="true">
+                <constraints nullable="false"/>
+            </column>
+        </addColumn>
+        <addColumn tableName="notebook">
```

Comment:
Seems like you could add in a few constraints, anything that is puiblished should automatically be checked as everpublic, otherwise it would be inconsistent.